python/pysrc/kaskada/_timestream.py

```diff
@@ -321,6 +321,14 @@ def hash(self) -> Timestream:
         """
         return Timestream._call("hash", self)
 
+    def len(self) -> Timestream:
+        """Return a Timestream with the length of input string.
```

Comment:
May want this to support lists as well. Not sure if we need to do anything *now*.